spring-boot/proxy/src/main/java/eu/coatrack/proxy/metrics/MetricsTransmitter.java

```diff
@@ -128,7 +128,7 @@ private void transmitToYggAdmin(eu.coatrack.api.Metric metricToTransmit) {
             */
 
         } catch (Exception e) {
-            log.error("Exception when communicating with CoatRack admin server", e);
+            log.error("An unexpected error occurred when your CoatRack gateway contacted the central CoatRack portal in order to send the statistics/metrics from this call: please assure that your CoatRack gateway has a network connection and can reach https://coatrack.eu", e);
```

Comment:
Please replace the hard-coded URL by the one from the config, to allow it to display the correct URL in different environments.

Reply:
Done and working 
![image](https://user-images.githubusercontent.com/47450617/92481330-80914e80-f1e6-11ea-91ac-8bb190ab5a26.png)